src/interfaces/IPoolManager.sol

```diff
@@ -122,11 +122,11 @@ interface IPoolManager is IProtocolFees, IERC6909Claims, IExtsload, IExttload {
     /// @param key The pool to modify liquidity in
     /// @param params The parameters for modifying the liquidity
     /// @param hookData Any data to pass to the callback, via `IUnlockCallback(msg.sender).unlockCallback(data)`
-    /// @return callerDelta The balance delta of the caller of modifyLiquidity. This is the total of both principal and fee deltas.
+    /// @return callerDeltas The balance delta of the caller of modifyLiquidity. This is the total of both principal and fee deltas.
     /// @return feeDelta The balance delta of the fees generated in the liquidity range. Returned for informational purposes.
```

Comment:
feeDeltas

Reply:
and balance deltas

Reply:
can you actually change feeDelta -> feesAccrued to match the named return param in PoolManager.sol